src/components/FilterOptions/ReasonFilterOptions/TriggersAccordion/TriggersPreview.tsx

```diff
@@ -0,0 +1,24 @@
+import { Preview } from "components/Preview"
+import { BailCodes } from "utils/bailCodes"
+
+interface BailCodeLineProps {
+  bailCode: string
+  bailName: string
+}
+
+const BailCodeLine = ({ bailCode, bailName }: BailCodeLineProps) => {
+  return <li>{`${bailCode} - ${bailName}`}</li>
+}
+
+const TriggersPreview = () => {
+  return (
+    <Preview className={"govuk-!-margin-top-2 govuk-!-margin-bottom-6"}>
+      <p>{"Included triggers:"}</p>
+      {Object.entries(BailCodes).map(([bailCode, bailName]) => (
```

Comment:
If you add a <ul> tag around the list it will be indented similar to the design
```suggestion
            <ul>
        {Object.entries(BailCodes).map(([bailCode, bailName]) => (
          <BailCodeLine key={bailCode} bailCode={bailCode} bailName={bailName} />
        ))}
      </ul>
```

![image](https://github.com/ministryofjustice/bichard7-next-ui/assets/22743709/21f514ba-50a2-4270-b210-6c2b98cf8170)